devel/asio/Portfile

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+PortSystem                  1.0
+PortGroup                   github 1.0
+
+github.setup                chriskohlhoff asio 1.12.0 asio-
+git.branch                  [strsed [join ${github.tag_prefix}]${github.version}[join ${github.tag_suffix}] {g/\./-/}]
```

Comment:
Might be an error on my part somewhere else on the portfile but it's needed because the distfile had a sort-of unexpected name for some reason.

Reply:
I figured it out. This was only needed for the fetch phase. The real tag is `asio-1-12-0` rather than `asio-1.12.0` as the `github.setup` suggests. I tried to remove the line after I already had it on my local drive which worked fine, but it failed on Travis. I then changed it to `1-12-0` and used a slightly simpler expression to fix the version number afterwards.